cli/src/base.ts

```diff
@@ -538,29 +565,11 @@ export abstract class BaseCommand<T extends typeof Command> extends Command {
     }
   }
 
-  async obtainKey() {
-    const { decision } = await this.prompt({
-      type: 'select',
-      name: 'decision',
-      message: 'Do you want to use an existing API key or create a new API key?',
-      choices: [
-        { title: 'Create a new API key in browser', value: 'create' },
-        { title: 'Use an existing API key', value: 'existing' }
-      ]
-    });
-    if (!decision) this.exit(2);
-
-    if (decision === 'create') {
-      return createAPIKeyThroughWebUI();
-    } else if (decision === 'existing') {
-      const { key } = await this.prompt({
-        type: 'password',
-        name: 'key',
-        message: 'Existing API key:'
-      });
-      if (!key) this.exit(2);
-      return key;
-    }
+  async obtainKey(web?: string) {
+    const currentProfile = await this.getProfile();
+    const domain = web || currentProfile?.web || 'https://app.xata.io';
+
+    return await loginWithWebUI(domain);
```

Comment:
We no longer give them the option to create a new API key during login, it's done automatically on login now?